Makefile

```diff
@@ -71,5 +71,15 @@ check-docs:
 	@echo ">> checking formatting and local/remote links"
 	$(MDOX_BIN) fmt --soft-wraps --check -l --links.validate.config-file=$(MDOX_VALIDATE_CONFIG) **/*.md
 
+.PHONY: deepcopy-gen
+deepcopy-gen:
```

Comment:
```suggestion
deepcopy-gen: deepcopy-gen-install
```

Reply:
I thought about it. The thing for me was that every time we run the target it will run the installation, and that's something we don't do for other targets. 

I have zero strong opinions on this, I guess this is something that we will hardly ever touch that we can just add the dependant as you say.